.travis.yml

```diff
@@ -3,13 +3,15 @@ node_js:
   - "0.11"
   - "0.10"
   - "0.8"
+  - "iojs"
+  - "iojs-v2.0.0"
+  - "iojs-v1.8.1"
+  - "iojs-v1.0.4"
 matrix:
   allow_failures:
-    - node_js: "0.11"
+    - node_js: "0.8"
 
 before_install:
-  # Upgrade `npm` to avoid semver issues
-  - npm install npm@1.x.x -g
```

Comment:
If we are deleting `npm install`, then we should delete the `before_install` key as well